plugin/src/lib/hooks/collections/afterChange.ts

```diff
@@ -242,7 +242,7 @@ const performAfterChange = async ({
       doc: previousDoc,
       fields: localizedFields,
     });
-    Object.keys(currentCrowdinHtmlData).forEach(async (name) => {
+    await Promise.all(Object.keys(currentCrowdinHtmlData).map(async (name) => {
```

Comment:
Highlighting this change as it is an important fix for tests and general robustness.